infra/gcp/terraform/kubernetes-public/k8s-kettle.tf

```diff
@@ -72,38 +72,6 @@ resource "google_bigquery_dataset_iam_policy" "prod_kettle_dataset" {
   policy_data = data.google_iam_policy.prod_kettle_dataset_iam_policy.policy_data
 }
 
-
-// Service account dedicated for BigQuery Data Transfer from BQ dataset k8s-gubernator:builds
-// TODO: remove when kettle migration is over
```

Comment:
Migration is over.

https://github.com/kubernetes/k8s.io/issues/789
https://github.com/kubernetes/k8s.io/issues/787
https://github.com/kubernetes/k8s.io/issues/1308